apps/electron/tsconfig.json

```diff
@@ -11,7 +11,8 @@
     "outDir": "dist",
     "moduleResolution": "node",
     "resolveJsonModule": true,
-    "noImplicitOverride": true
+    "noImplicitOverride": true,
+    "verbatimModuleSyntax": false
```

Comment:
Why

Reply:
verbatimModuleSyntax cannot be used with `export const` from an external module.